extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -687,23 +682,50 @@ export abstract class AzureAuth implements vscode.Disposable {
 			booleanResult: false
 		};
 
+		const cancelAndAuthenticate: ConsentMessageItem = {
+			title: localize('azurecore.consentDialog.authenticate', "Cancel and Authenticate"),
+			isCloseAffordance: true,
+			booleanResult: true
+		};
+
 		const dontAskAgainItem: ConsentMessageItem = {
 			title: localize('azurecore.consentDialog.ignore', "Ignore Tenant"),
 			booleanResult: false,
+			action: async (tenantId: string) => {
+				return await confirmIgnoreTenantDialog();
+			}
+		};
+
+		const confirmIgnoreTenantItem: ConsentMessageItem = {
+			title: localize('azurecore.confirmIgnoreTenantDialog.IConfirm', "I confirm"),
```

Comment:
Usually button text doesn't have the subject in it, I think "Confirm" is cleaner

```suggestion
			title: localize('azurecore.confirmIgnoreTenantDialog.confirm', "Confirm"),
```